### DISCUSSION TITLE
How to know which axis is locked when setting axis to lock

### DISCUSSION THREAD

**Discussion by xu33** [pmndrs/use-gesture#625] [Q&A] [answered]
To differentiate between horizontal and vertical movements and handle them differently, how can we determine which direction is locked during the current drag?

`
const bind = useDrag(
    ({ down, movement: [mx, my] }) => {
     // if axis is lock to x, scroll the content horizontally
    // if axis is lock to y, move the container up  (like a drawer)
    },
    { axis: 'lock' }
  )
`

**Comment by dbismut** [comment, marked as answer]
Not sure I understand. If the axis locked is `x`, `my` will be 0.

**Reply by xu33** [reply]
Thank you, i will try this!
